pkg/filesystem/file.go

```diff
@@ -2,6 +2,13 @@ package filesystem
 
 import (
 	"io"
+	"time"
+)
+
+var (
+	// DeterministicFileModificationTimestamp is a fixed timestamp set to
```

Comment:
This comment is somewhat tautological. It describes what it is, as opposed to describing the intent. What are your thoughts on something like this?

> DeterministicFileModificationTimestamp is a fixed timestamp that can be provided to Directory.Chtimes() to give files deterministic modification times. It is used by bb_worker to ensure that all files in the input root of a build action have the same modification time. This is needed to make certain kinds of build actions (most notably Autoconf scripts) succeed.
>
> 2000-01-01T00:00:00Z was chosen, because it's easy to distinguish from genuine timestamps. 1970-01-01T00:00:00Z would be impractical to use, because it tends to cause regressions in practice. Examples:
>
> https://bugs.python.org/issue34097                                            
> https://gerrit.wikimedia.org/r/#/c/mediawiki/core/+/437977/ 

Could you also move this constant into directory.go? I guess it makes more sense to place it there, because Chtimes() is also declared there.